src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -34,95 +34,10 @@ def get_simulant_id_maps(
     """
     if column_name != "simulant_id":
         raise ValueError(f"Expected `simulant_id`, got `{column_name}`")
-
-    ssn_map = get_ssn_map(obs_data, column_name, artifact)
-
-    return ssn_map
-
-
-def generate_itins(
```

Comment:
We are using the artifact data instead of the function now. There is also no concept of "making up" ITINs like there is for SSNs so we really don't need this